config/config.go

```diff
@@ -20,7 +20,8 @@ type Config struct {
 	Services                     json.RawMessage            `json:"services"`
 	Labels                       map[string]string          `json:"labels"`
 	Discovery                    json.RawMessage            `json:"discovery"`
-	Bundle                       json.RawMessage            `json:"bundle"`
+	Bundle                       json.RawMessage            `json:"bundle"` // Deprecated: Use `bundles` instead
```

Comment:
Would it be worthwhile to log something when `bundle` is used?

Reply:
I thought about adding a log for that, some warning that updating to `bundles` should happen, but didn't see a good place to put it without adding logrus to packages we don't really want it in. Maybe as a follow on we can figure out a good place for it, but I think it might take some refactoring.